Ui/Component/Listing/Column/TrackAndTrace.php

```diff
@@ -101,10 +102,14 @@ public static function getTrackAndTraceLinksAsHtml(Order $order): string
         }
 
         foreach($trackNumbers as $trackNumber) {
-            if (self::VALUE_EMPTY === $trackNumber) {
+            if (self::VALUE_EMPTY === $trackNumber || null === $trackNumber) {
                 $html .= '-<br/>';
                 continue;
             }
+            if (self::VALUE_PRINTED === $trackNumber) {
```

Comment:
switch/case? En waarom zijn we hier eigenlijk kale strings aan het vergelijken? 🥺

Reply:
Zelfde als bij 'concept'...